### PR TITLE
Insync: mark broken

### PR DESCRIPTION
**Pull request by InLaw**
###### Things done

package url is not existing anymore 
Users should be aware of the unusable package 

set 
- broken = "url doesn't exist anymore"

---

@benley

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
> package url is not existing anymore
> Users should be aware of the unusable package

Why not update the package and then use web.archive.org to preserve it in the future?

**Comment by InLaw** [comment]
> Why not update the package and then use web.archive.org to preserve it in the future?

update issues:
https://discourse.nixos.org/t/update-insync-auto-patchelf-rm-lib-issues-e-g-libglx-so-0/22064

web.archive.org - sure, if that is best practice (is there already a (nixos)-process/workflow for it?)

**Comment by SuperSandro2000** [comment]
Just input the URL into https://web.archive.org/save and hit Save page and you get an archive URL.

**Comment by InLaw** [comment]
(the process is quite simple and steps are clear and predictable. An automated process [that triggers a saving of the new url] would be helpful - because that is "an issue" for all users, no?)

since I'm not sure when I will find time for investigating the issues in installPhase, it would be best to make the current state transparent for all users 
(because they don't know about it yet. https://discourse.nixos.org/t/old-unusable-apps-in-nixpkgs/22083/8 - sometimes state is bad but here state in nixos could be a good think [temporarily] for some users -)

**Comment by SuperSandro2000** [comment]
> (the process is quite simple and steps are clear and predictable. An automated process [that triggers a saving of the new url] would be helpful - because that is "an issue" for all users, no?)

I don't think we have something like that but it would be cool.

**Review by SuperSandro2000 — COMMENTED** [review]
Please follow the [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes) when naming your commits.

Also can you add a short comment why the packages are marked broken? and ideally the lines would be moved last in meta.

File: pkgs/applications/networking/insync/default.nix [hunk 1]
```diff
@@ -39,5 +39,6 @@ stdenv.mkDerivation rec {
 
      There is a 15-day free trial, and it is a paid application after that.
     '';
+    broken = true;
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/insync/default.nix:42]:
```suggestion
    # download URL removed
    broken = true;
```

File: pkgs/applications/networking/insync/v3.nix [hunk 2]
```diff
@@ -80,5 +80,6 @@ stdenv.mkDerivation rec {
 
      There is a 15-day free trial, and it is a paid application after that.
     '';
+    broken = true;
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/insync/v3.nix:83]:
```suggestion
    # download URL removed
    broken = true;
```

**Comment by benley** [comment]
Thanks.  Hopefully someone can resurrect this package later on, because it was useful when it worked.

**Pull request merged**